test/petab/test_petab_import.py

```diff
@@ -190,18 +193,24 @@ def test_max_sensi_order():
     )
 
     # fix max_sensi_order to 1
-    objective = problem.create_objective(max_sensi_order=1)
+    objective = importer.create_problem(
+        objective=importer.create_objective(max_sensi_order=1)
+    ).objective
```

Comment:
This might be a minor thing, but i find this a really unintuitive way of creating the objective function correctly.

Reply:
The problem is, that petab fixed parameters are only handled in `create_problem`, but not in `create_objective`. Maybe this could/should be changed. Not completely sure what might rely on the current behaviour.

Reply:
well both the importer and the problem should be aware of fixes parameters, so I agree that one should naturally expect `problem.create_objective`, `importer.create_objective` and `importer.create_problem().objective` to be equivalent and that passing `importer.create_objective(max_sensi_order=1)` to `importer.create_problem` isn't necessary.